biz.aQute.bndlib/src/aQute/bnd/osgi/Descriptors.java

```diff
@@ -120,10 +121,26 @@ public String getPath() {
 			return binaryName;
 		}
 
+		/**
+		 * @return <code>true</code> if this is a "java." package.
+		 */
 		public boolean isJava() {
 			return java;
 		}
 
+		/**
+		 * @return <code>true</code> if this package is part of the JDK for the
+		 *         given ee.
+		 */
+		public boolean isJDK(EE ee) {
```

Comment:
I'm having second thoughts about whether or not to make a method here or simply inline the check in `Analyzer`. Adding it here makes it part of public API which adds to the maintenance overhead. Further,  it's also coupling this API to another package which is not otherwise coupled to this package.

It might be safer to inline it as we can always add it to public API later if there is a demand, but it's hard to remove it from public API once it is there.

Reply:
I will move as private in `Analyzer`.

Reply:
done in https://github.com/bndtools/bnd/pull/6508/commits/2cd573d423e3fff52238f9bcbd06bfc033cc629e